### PR TITLE
chore: remove detox

### PR DESCRIPTION
**Pull request by Belema**
It appears that we are not using `detox` anymore.

This PR removes the `functional-tests:*` scripts that are unused.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -122,10 +122,6 @@ running `yarn ios`. You will need to terminate the server that is run with
 this as fructose will run its own at the time of testing. As long as you do not
 delete the app, you will only have to build the app once.
 
-If you run `npm run functional-test:ios` in the root directory it will run
- fructose tests for all of the components. This will similarly work for android
```
Comment (colincclark) [inline .github/CONTRIBUTING.md]: is it fructose or detox you are intending to remove? There are more references to both of these tools in this file....

File: bitrise.yml [hunk 2]
```diff
@@ -25,33 +25,6 @@ workflows:
             yarn ios -- --simulator "iPhone 7"
```
Comment (L0wry) [inline bitrise.yml]: We can actually delete this file as we dont use the bitrise.yml

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (L0wry) [reply .github/CONTRIBUTING.md]: we can remove references to detox too

**Comment by L0wry** [comment]
solid, Should also be able to remove the folder `lib/functional_tests`

**Comment by craigbilner** [comment]
WIP

File: bitrise.yml (hunk 2 above)
Comment (colincclark) [reply bitrise.yml]: will do

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/82e7d31bce960d3969c8bd239e00b7e736d7cdca/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/a56258eca590901d10e374ab73f0226f99d0d323/index.html

**Pull request closed without merging**